addons/mail/static/src/js/chatter.js

```diff
@@ -31,6 +31,7 @@ var Chatter = Widget.extend({
         discard_record_changes: '_onDiscardRecordChanges',
         reload_attachment_box: '_onReloadAttachmentBox',
         reload_mail_fields: '_onReloadMailFields',
+        need_refresh_suggested_partner: '_refresh_suggested_partner',
```

Comment:
entries in `custom_events` and `events` should follow alphabetical order of handler method name (`events` does not seem to be ordered, but you can ignore this in your diff).